status-im/status-react
# 📜Terms of service UI changes

fixes #12122 
fixes #12208 
fixes #12206
fixes #12207 

### Summary
- Link to Status website for for privacy policy and terms of service
- Terms of service in about app sections
- Web view instead of browser redirect for ToS and privacy policy at all places
- Opt-in checkbox for new users on onboarding screen
- Opt-in screen for existing users

### Testing notes
Might break onboarding because the user needs to accept ToS (check checkbox). Until checkbox is checked, the "Get started" button is disabled

#### Platforms
- Android
- iOS

#### Areas that maybe impacted
- Onboarding

### Steps to test

ToS opt-in for new users
- Delete all accounts
- On the carousel screen, you will see a checkbox
- The "Get started" button should be disabled until the checkbox is checked
- ToS and privacy policy should open web views linking to Status website

ToS in Settings
- Go to profile > About app
- You should see an item of Terms of service
- Clicking that should open a web view to Status website

ToS opt-in for existing users
- If the app already has some multi accounts and the welcome opt-in was never displayed, the user will be redirected to an opt-in screen (#12207). 
- This screen should not be visible:
  - For new downloads 
  - For users who opted-in but deleted all multi accounts

- This screen should be visible:
  - For users who have one or more multiaccounts but never opted-in during onboarding (at carousel screen)
